applications/holoscan_ros2/vb1940/Dockerfile

```diff
@@ -0,0 +1,105 @@
+# syntax=docker/dockerfile:1
+
+# SPDX-FileCopyrightText: Copyright (c) 2022-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+############################################################
+# Build the vb1940 image with ROS2 and hololink dependencies
+############################################################
+ARG BASE_IMAGE=nvcr.io/nvidia/clara-holoscan/holoscan:v3.5.0-dgpu
+FROM ${BASE_IMAGE} AS vb1940
+
+# Install basic dependencies
+RUN apt update && apt install -y \
+    curl \
+    gnupg \
+    lsb-release \
+    software-properties-common \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install git and openssh-client so we can clone the hololink repository
+RUN apt update && apt install -y \
+    git \
+    openssh-client \
+    && rm -rf /var/lib/apt/lists/*
+
+# Add ROS2 Jazzy repository
+RUN curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/ros2.list > /dev/null
+
+# Install ROS2 Jazzy and comprehensive Vulkan support
+RUN apt update && apt install -y \
+    ros-jazzy-desktop \
+    ros-jazzy-ros-base \
+    vulkan-tools \
+    vulkan-validationlayers \
+    libvulkan1 \
+    && rm -rf /var/lib/apt/lists/*
+
+# Set ROS2 environment variables directly
+ENV ROS_DISTRO=jazzy
+ENV ROS_VERSION=2
+ENV ROS_PYTHON_VERSION=3
+ENV CMAKE_PREFIX_PATH=/opt/ros/jazzy:/opt/nvidia/holoscan
+ENV AMENT_PREFIX_PATH=/opt/ros/jazzy
+ENV LD_LIBRARY_PATH=/opt/ros/jazzy/lib:$LD_LIBRARY_PATH
+ENV PATH=/opt/ros/jazzy/bin:$PATH
+ENV PKG_CONFIG_PATH=/opt/ros/jazzy/lib/pkgconfig
+
+# Set comprehensive Vulkan environment variables for Holoviz
+ENV VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/nvidia_icd.json
+ENV VK_LAYER_PATH=/usr/share/vulkan/explicit_layer.d
```

Comment:
I'm not sure this should typically be necessary in order to use Holoviz? Would expect at least `nvidia_icd.json` to be automatically mounted by `--runtime=nvidia` and not explicitly specified inline